serverless/index-serverless-project-settings.asciidoc

```diff
@@ -25,3 +27,16 @@ include::./pages/transforms.asciidoc[leveloffset=+3]
 include::./pages/integrations.asciidoc[leveloffset=+2]
 
 include::./pages/fleet-and-elastic-agent.asciidoc[leveloffset=+2]
+
+include::./pages/manage-org.asciidoc[leveloffset=+2]
+
+include::./pages/manage-your-project.asciidoc[leveloffset=+2]
+include::./pages/ml-nlp-auto-scale.asciidoc[leveloffset=+3]
+include::./pages/manage-your-project-rest-api.asciidoc[leveloffset=+3]
+
+include::./pages/manage-billing.asciidoc[leveloffset=+2]
```

Comment:
We could shrink this whole section into a single page to focus on serverless and link out to the cloud docs for shared Elastic Cloud concepts